packages/emails/src/send-email.tsx

```diff
@@ -5,6 +5,12 @@ import { createTransport, Transporter } from "nodemailer";
 import type Mail from "nodemailer/lib/mailer";
 import React from "react";
 
+import { sendgridTransport } from "nodemailer-sendgrid-transport";
+import { mandrillTransport } from "nodemailer-mandrill-transport";
+import { mailgunTransport } from "nodemailer-mailgun-transport";
+import { sendinblueTransport } from "nodemailer-sendinblue-transport";
+import { mailjetTransport } from "nodemailer-mailjet-transport";
```

Comment:
I think importing these on demand as you had done with require was a good idea to keep the initial bundle size smaller.

Let's try using import inline inside the switch statement instead.

```
const { sendgridTransport } = await import("nodemailer-sendgrid-transport");
```